src/app/components/forms/verified-account/verified-account.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+	<app-form name="verifiedAccountForm">
+		<app-form-group name="application" :label="$gettext(`Application Text`)">
+			<app-form-control-textarea :maxlength="maxChars" rows="2" :disabled="!canSubmit" />
+			<app-form-control-errors />
+			<div v-if="!existingApplication" class="help-block">
+				Provide us with a short explanation as to why you think your account should be verified!
```

Comment:
Assuming no translate block because we do require english for this section?

Reply:
Correct.